client/core/bond.go

```diff
@@ -123,6 +283,14 @@ func (c *Core) refundExpiredBonds(ctx context.Context) {
 				continue
 			}
 
+			// Here we may either refund or renew the bond depending on target
+			// tier and timing. Direct renewal (refund and post in one) is only
+			// useful if there is insufficient reserves or the client had been
+			// stopped for a while. Normally, a bond becoming spendable will not
+			// coincide with the need to post bond.
+			//
+			// TODO: if mustPost > 0 { wallet.RenewBond(...) }
```

Comment:
Is it possible to spend multiple expired bonds in one renewal, rather than renew individually?

Reply:
It would be possible yes.  Timing would be even less likely in practice though.  Putting this PR into work on testnet where the lock times are much longer helps get a feel for more real life cadences where there's a fair amount of bond overlap and very little coincidence of <it's time to post> with <it's time to refund>.  The only way to make that happen more readily is to simply not refund until <it's time to post>.  I'm reluctant to leave funds locked in bonds longer than needed, but I'm considering it.

Working through some concepts that will help decide about this in a `bond-reserves` branch for a PR very soon.